BREAKING_CHANGES.md

```diff
@@ -0,0 +1,11 @@
+# Import conductor V2
```

Comment:
I like this file - I will also copy the content into the Release notes here on GitHub